### ISSUE TITLE
Make xml-rpc an organization project

### ISSUE THREAD

**Issue by grettke** [xml-rpc-el/xml-rpc-el#10]
According to MELPA, xml-rpc is used by confluence, debpaste, dokuwiki, mentor, org2blog, tracwiki-mode, and weblogger (if not more). In my case I maintain Org2Blog for which XML-RPC is critical.

I'd like to invest my time and effort in co-maintaining this package. I've got skin in the game on this and that is how I can pay up.

If you are willing to have me as a co-maintainer than the easiest way to move forward would be to set up this project as an [Organization](https://help.github.com/en/github/getting-started-with-github/types-of-github-accounts#organization-accounts). For example that works well for [Org2Blog](https://github.com/orgs/org2blog/people). Given that it is so hard to find maintainers in the first place, it is better to have more than one person who can mange the repo. In particular when people die or quit participating entirely that still leaves someone to find new maintainers or whatnot. Additionally, people do come and go on projects. Having a person familiar with the project though inactive can ease those transitions.

Either way let me know what you think.

Thanks and have a great day.

**Comment by hexmode** [comment]
I love this idea.  I'm not a big user of this project right now (though I used to be) so I'm completely open to giving up control -- I'm grateful for your offer.

Do you have a suggestion for the organisation name?

**Comment by grettke** [comment]
> I love this idea.

Cool!

> I'm not a big user of this project right now (though I used to be) so I'm completely open to 
> giving up control -- I'm grateful for your offer.

Thank you for the kind words. 

Thank you for the package: it is greatly appreciated.

When I started working on Org2Blog as the maintainer even though the creator no longer really uses it anymore he is still always around. We've had great results with the organization approach. 

> Do you have a suggestion for the organisation name?

I suggest the organization name "xml-rpc-el" because the name distinguishes itself from all other xml-rpc related organizations (assuming there are any) and it is also simple being the same name as the repo itself. 

That is what we did with org2blog: the organization name is org2blog and the repo name is org2blog and it worked well.
